<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Set_indiviual_time</class>
 <widget class="QDialog" name="Set_indiviual_time">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>533</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1021</width>
    <height>533</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1021</width>
    <height>533</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pBtnUseOnOff1_set_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>501</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnUseOnOff1_set_18">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>491</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>저장</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_set_flow_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>1001</width>
     <height>331</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>41</width>
      <height>331</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_17">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>41</width>
      <height>331</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_18">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>41</width>
      <height>331</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_indvd_time_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_indvd_time_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_indvd_time_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_indvd_time_12">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_indvd_time_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_indvd_time_8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_indvd_time_13">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_indvd_time_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_indvd_time_9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_indvd_time_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_indvd_time_14">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>14</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>13</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_39">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>12</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_40">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>11</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_indvd_time_5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_indvd_time_10">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>140</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_indvd_time_15">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>250</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
   <widget class="Line" name="line_19">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>41</width>
      <height>331</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>110</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>220</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>15</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_indvd_time_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>_ _ : _ _</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblDispDate_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1001</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>개별시간</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pBtnUseOnOff1_set_17</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBtnUseOnOff1_set_18</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_indvd_time_1</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_time_part()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_indvd_time_2</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_time_part()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_indvd_time_3</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_time_part()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_indvd_time_4</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_time_part()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>762</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>778</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_indvd_time_5</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_time_part()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>947</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>957</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_indvd_time_6</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_time_part()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_indvd_time_7</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_time_part()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_indvd_time_8</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_time_part()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_indvd_time_9</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_time_part()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>634</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>1014</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_indvd_time_10</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_time_part()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>977</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>1016</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_indvd_time_11</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_time_part()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_indvd_time_12</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_time_part()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_indvd_time_13</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_time_part()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_indvd_time_14</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_time_part()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>632</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_indvd_time_15</sender>
   <signal>clicked()</signal>
   <receiver>Set_indiviual_time</receiver>
   <slot>set_indvd_time_part()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>848</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>864</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>set_cancel()</slot>
  <slot>set_indvd_save()</slot>
  <slot>set_indvd_time_part()</slot>
 </slots>
</ui>
